PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -476,11 +476,12 @@ protected function exchange_bind_ok($reader)
      * @param string $destination
      * @param string $source
      * @param string $routing_key
+     * @param bool $nowait
      * @param array $arguments
      * @param int $ticket
      * @return mixed
      */
-    public function exchange_unbind($destination, $source, $routing_key = '', $arguments = null, $ticket = null)
+    public function exchange_unbind($destination, $source, $routing_key = '', $nowait = false, $arguments = null, $ticket = null)
```

Comment:
I made it this way because i don't think it's likely anyone could have been using the optional parameters as the implementation won't work as far as i can tell. Also it follows the order as described in the docs, and follows the established convention.